test/extended/baremetal/hosts.go

```diff
@@ -187,3 +188,37 @@ var _ = g.Describe("[sig-installer][Feature:baremetal][Serial] Baremetal platfor
 		o.Expect(found).To(o.BeFalse())
 	})
 })
+
+var _ = g.Describe("[sig-installer][Feature:baremetal] A baremetal deployment without a provisioning network should", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLI("baremetal")
+	skipIfNotBaremetal(oc)
+	dc := oc.AdminDynamicClient()
+	skipIfProvisioningNetworkSet(dc)
+
+	g.By("show the Provisioning Network as 'Disabled'")
+	o.Expect(getProvisioningNetwork(dc)).To((o.BeEquivalentTo("Disabled")))
+
+	g.It("not allow setting the ProvisioningNetwork to \"Managed\" with invalid values", func() {
```

Comment:
nit: \`not allow setting the ProvisioningNetwork to "Managed" with invalid values\`?

Reply:
mmm I need to escape the double quotes here, or I can change that to single quotes

Reply:
I mean, if you use "`" instead double quotes, you don't need to escape double quotes.

Reply:
oh, right! ok